index.html

```diff
@@ -50,31 +50,31 @@ <h1>
         </p>
         <ul class="social-ul">
           <li>
-            <a href="#"><img src="Images/twitter.svg" alt="twitter" /></a>
+            <a href="#"><img src="Images/twitter.svg" alt="This is twitter button" /></a>
           </li>
           <li>
-            <a href="#"><img src="Images/Linkedin.svg" alt="Linkedin" /></a>
+            <a href="#"><img src="Images/Linkedin.svg" alt="This is linkedin button" /></a>
           </li>
           <li>
-            <a href="#"><img src="Images/medium.svg" alt="medium" /></a>
+            <a href="#"><img src="Images/medium.svg" alt="This is medium button" /></a>
           </li>
           <li>
-            <a href="#"><img src="Images/github.svg" alt="github" /></a>
+            <a href="#"><img src="Images/github.svg" alt="This is github button" /></a>
           </li>
           <li>
-            <a href="#"><img src="Images/angellist.svg" alt="angellist" /></a>
+            <a href="#"><img src="Images/angellist.svg" alt="This is angellist button" /></a>
           </li>
         </ul>
         <a href="#">
-          <img src="Images/Disabled.svg" alt="GoDown" />
+          <img src="Images/Disabled.svg" alt="This is go down button" />
         </a>
       </header>
     </section>
     <section id="my-recent-works">
       <h2 class="section-header">My Recent Works</h2>
       <div id="cards-grid-container">
         <div class="card">
-          <img class="card-img" src="Images/SnapshootPortfolio.png" alt="" />
+          <img class="card-img" src="Images/SnapshootPortfolio.png" alt="This is project 1 image" />
```

Comment:
- [ ] I think you could make the same changes here. You could use something like "Project 1 image" or "Project 1" :wink:
**Please, update all the similar occurrences** :pushpin:

Reply:
Done